translate/src/modules/user/components/SignIn.css

```diff
@@ -6,7 +6,7 @@
   background: transparent;
   border: 1.5px solid var(--status-translated);
   border-radius: 2px;
-  color: white;
+  color: #000;
```

Comment:
Please note that Pontoon uses themes, so rather than hardcoding color values, we use variables. 

See https://github.com/mozilla/pontoon/blob/main/pontoon/base/static/css/style.css#L665-L681 for the values that are used in all views but the translate view.